android_mobile/src/main/java/com/alexstyl/specialdates/events/namedays/NamedayLocale.java

```diff
@@ -11,6 +11,7 @@
     ROMANIAN("ro", false, R.string.Romanian, R.raw.ro_namedays),
     RUSSIAN("ru", false, R.string.Russian, R.raw.ru_namedays),
     LATVIAN("lv", false, R.string.Latvian, R.raw.lv_namedays),
+    LATVIANEXTENDED("lvext", false, R.string.LatvianExt, R.raw.lv_ext_namedays),
```

Comment:
The `shortcode` (first constructor parameter) is the real shortcode of a country (I found them from https://stackoverflow.com/questions/7973023/what-is-the-list-of-supported-languages-locales-on-android). 

Let's have this one use `lv` instead even if it is the same as Latvian. They do refer to the same country don't they?